src/frontends/common/include/openvino/frontend/extension.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//
+// Includes all extensions.
+//
+
+#pragma once
+
+#ifdef IN_OV_CORE_LIBRARY
```

Comment:
@ilyachur should I use another definition here?

Reply:
In this case we need to use common define inside the OV libraries.